client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+            "selectGotoTarget": "Setzte einen Zielpunkt:",
+            "selectZones": "Auswahl der Zonen zur Reinigung:",
+            "selectMapToLoad": "Auswahl des Speicherplatzes zum Laden der Karte:",
+            "emptyZoneChosen": "Es wurde eine leere Zone gewählt. Bitte zuerst die Zone einrichten.",
+            "tooManyZones": "Es wurde versucht {{count}} Unterzonen zu wählen. Es werden maximal 5 unterstützt.",
+            "chooseMapName": "Titel zum Speichern der Karte wählen:",
+            "invalidMapName": "Der Titel darf nur Buchstaben, Zahlen, Leerzeichen und Bindestriche enthalten.",
+            "slotAlreadyTaken": "Speicherplatz #{{slot}} ist bereits in Benutzung",
+            "nameAlreadyTaken": "Name \"{{name}}\" wurde schon verwendet",
+            "noMapsStored": "Keine gespeicherte Karten!",
+            "slotNo": "Speicherplatz #",
+            "confirmMapDelete": "Soll \"{{name}}\" wirklich gelöscht werden?",
+            "wait": "Warten",
+            "waitMapTicks": "Kartenersetzungsroutinen laufen. Bitte weitere {{seconds}} Sekunden warten...",
+            "waitMapFinished": "Beendet",
+            "waitMapEncourage": "Angeforderte Routinen sollten nun beendet sein.",
+            "mapStoredOK": "Karte erfolgreich gespeichert!",
+            "mapDeletedOK": "Gespeicherte Karte erfolgreich gelöscht!",
+            "mapRestoredOK": "Karte erfolgreich wiederhergestellt!"
+        },
+        "fanspeedPresets": {
+            "whisper": "Flüstern",
+            "quiet": "Leise",
+            "balanced": "Ausgeglichen",
+            "turbo": "Turbo",
+            "max": "Maximal",
+            "mop": "Wischen"
+        }
+    },
+    "map": {
+        "status": "Status",
+        "battery": "Batterie",
+        "connecting": "Verbinden...",
+        "statusText": "{{status, lowercase}}",
+        "errorText": "{{error, lowercase}}",
+        "confirmFullCleaning": "Zonenreinigung kann nicht gestartet werden: Keine Zonen spezifiziert. Soll stattdessen eine komplette Reinigung gestartet werden?",
+        "cannotResume": "Nichts zur Wiederaufnahme!",
+        "tooManyZones": "Es können nicht mehr als 5 Zonen auf der Karte gesetzt werden."
+    },
+    "zones": {
+        "headerZones": "Eingestellte Zonen",
+        "headerGotoLocations": "Eingestellte Zielpositionen",
+        "headerForbiddenMarkers": "Verbotsmarkierungen",
+        "noZonesConfigured": "Bisher wurden keine Zonen eingestellt.",
+        "noSpotsConfigured": "Bisher wurden keine Zielpunkte eingestellt.",
+        "editForbiddenZones": "Einstellen von Verbotszonen",
+        "noPersistentData": "Dauerhafte Daten sind abgeschaltet oder nicht verfügbar. Es können keine Verbotsmarkierungen bearbeitet werden.",
+        "addButton": "Hinzufügen",
+        "renameButton": "Umbenennen",
+        "deleteButton": "Löschen",
+        "newZoneName": "Zonenname spezifizieren ...",
+        "newSpotName": "Markierungsname spezifizieren...",
```

Comment:
to avoid mixing this up with the forbidden zone markers that are partially translated with "Markierung" and keep the reference to the "spot" button on the home page I suggest to use "Spotreinigungspunkt" or keep "Spot" in the name:
"enterSpotName": "Spotnamen festlegen..."

Reply:
Spot is a word i actually had to explain to my grandpa. So maybe it is not the best choice.

Reply:
IMHO, there should be a clear reference of everything regarding spot definitions to the "Spot" button on the Valtudo homepage, since otherwise it would be unclear to the user that the "Spot" button is used to move the robot to a spot and lets him clean there. Hence my attempts to use the same word throughout all corresponding settings.

Since "Punkt" or "Ziel" alone is not really descriptive on that button, and words with more than about 8 characters do not fit on small devices with the design being set as it is, I ended up reusing "Spot" since I did not find anything better... I remember the sentence "Licht aus - Spot an!" from a german TV show running from the early seventies for over ten years, so I considered this common knowledge by now ;-)

Reply:
Still, "Ziel" would be the correct definiton because it also is used to just move to the target point.
My robot also says when I move it "going to the target point" There is a difference here between Ziel and Spot. Due to the extended area on a spot cleanup.

Reply:
... That's it - "Spot_reinigung_" for that button! Thank you! For some I never attached the "Spot" label to the process but always connected it with the place. I should have used it again before doing the translation.
Manchmal sieht man den Wald vor lauter Bäumen nicht ^^ (sorry, don't know if this proverb exists in english and how it would be called there)

To conclude: as it is, I would suggest relabeling the "Spot" button even in the english version, but leave all the rest down here as it is. Resolved :-)